doc/source/whatsnew/v0.25.0.rst

```diff
@@ -427,6 +427,7 @@ Other
 ^^^^^
 
 - Removed unused C functions from vendored UltraJSON implementation (:issue:`26198`)
+- Removed usage of ``register`` keyword in C extension modules (:issue:`26263`)
```

Comment:
not necessary as the introduction was in 0.25 (u can add this issue number to the other note if u want)